Shared/Coordinator/MainCoordinator.swift

```diff
@@ -81,15 +81,18 @@ class MainCoordinator: Coordinator<Void> {
 
 
     func handle(deeplink: DeepLinkable) {
-        guard let string = deeplink.customMetadata["target"] as? String,
-              let target = DeepLinkTarget(rawValue: string)  else { return }
+        guard let target = deeplink.deepLinkTarget else { return }
+
+        self.deepLink = deeplink
```

Comment:
The deep link needs to be set so that the run conversation flow starts on the right conversation.